### PR TITLE
Disable the angular analyzer

### PR DESCRIPTION
**Pull request by kwalrath**
Contributes to #561.

/cc @matanlurey

### CODE REVIEW

**Comment by kwalrath** [comment]
Oh right, I'll have to update the tests again.

File: templates/web-angular/analysis_options.yaml [hunk 1]
```diff
@@ -2,8 +2,10 @@ analyzer:
   exclude: [build/**]
   errors:
     uri_has_not_been_generated: ignore
-  plugins:
-    - angular
+  # Angular plugin support is in beta. You're welcome to try it and report
+  # issues: https://github.com/dart-lang/angular_analyzer_plugin/issues
+  # plugins:
+    # - angular
```
Comment (chalin) [inline templates/web-angular/analysis_options.yaml]: Btw, you could make this just one line:

> # plugins: [angular]

(Sry, it hadn't occurred to me to mention this before).

Comment (kwalrath) [reply templates/web-angular/analysis_options.yaml]: Oh, that's nicer. :/

Comment (kwalrath) [reply templates/web-angular/analysis_options.yaml]: But I think I'll leave it as-is, for now.

**Comment by kwalrath** [comment]
I'll plan to publish stagehand Monday, unless someone really wants this sooner.

**Comment by kevmoo** [comment]
Monday is fine!

**Pull request merged**